Sage/carbon
# feat(card): describe card using design tokens - FE-4502

### Proposed behaviour

Describes the card component using design tokens.
Changes boxShadow color according to the design. Uses boxShadow050 tokens for non-interactive shadow and boxShadow100 tokens for interactive shadow.
Updates tests after the above changes.

### Current behaviour

Card use the theme styled-component property.
Current boxShadows: 
- non-interactive shadow: rgba(0, 20, 29, 0.2) 0px 3px 3px 0px, rgba(0, 20, 29, 0.15) 0px 2px 4px 0px,
- interactive shadow:  rgba(0, 20, 29, 0.2) 0px 5px 5px 0px, rgba(0, 20, 29, 0.1) 0px 10px 10px 0px

### Checklist




#### QA



### Testing instructions

The correct appearance can be checked in the component: card.
For theme: mint and aegean there should be no differences.
Use devtools to check if card use css variables.
